test/MsgPack.UnitTest/Serialization/VersioningTest.cs

```diff
@@ -63,23 +63,33 @@ private static void TestExtraFieldCore<T>( SerializationMethod method, EmitterFl
 				}
 				else
 				{
-					var packer = Packer.Create( stream, false );
-					packer.PackMapHeader( 5 );
+					const string String10 = "1234567890";	// packed as MinimumFixedRaw or Bin8
+					const string String40 = "1234567890123456789012345678901234567890"; // packed as Bin8 or Str8
+					var packer = Packer.Create( stream, compat, false );
+					packer.PackMapHeader( 7 );
 					packer.Pack( "Field1" );
 					packer.Pack( 1 );
-					packer.Pack( "Extra" );
-					packer.PackNull();
-					packer.Pack( "Field2" );
+					packer.Pack("Extra1");
+					packer.Pack(String40);
+					packer.Pack("Extra2");
+					packer.Pack(System.Text.ASCIIEncoding.Default.GetBytes(String40));
```

Comment:
Do you want to specify regular ASCIIEncoding here? There are 2 minor problems:
1. `ASCIIEncoding.Default` does NOT return ASCII encoding in many countriies (but they are usually compatible to ASCII for ascii charactors, so this is really trivial issue because `String40` only contains ascii chars).
1. `ASCIIEncoding` does not exist in WinRT, so this line breaks build.
Since `String40` has ASCII charactors only, so you can use `Encoding.UTF8.GetBytes(String40)` safely here.